Handler/Sites.hs

```diff
@@ -22,7 +23,7 @@ getSitesR = do
 
 postSitesR :: Handler Html
 postSitesR = do
-    userId <- requireAuthId
+    Entity userId user <- requireAuth
```

Comment:
I don't care about people hacking the form, myself. The effort required to craft a request (with an authenticated session) is enough of a deterrent IMO. Could be worth, at some point, tracking if enough people are curiously over quota to warrant locking down.

If I understand it correctly, there's no race condition now -- but there would be, if we added the check here. Is that right?